cmd/livepeer_cli/wizard_transcoder.go

```diff
@@ -41,8 +40,14 @@ func (w *wizard) promptOrchestratorConfig() (float64, float64, *big.Int, string)
 	fmt.Printf("Enter fee share percentage (default: 5) - ")
 	feeShare = w.readDefaultFloat(5.0)
 
-	fmt.Printf("Enter price per segment in Wei (default: 1) - ")
-	pricePerSegment = w.readDefaultBigInt(big.NewInt(1))
+	fmt.Printf("Enter a transcoding base price in wei per pixels")
```

Comment:
Missing new line here - we should be able to just use `fmt.Println()`

Reply:
fixed in 894375e